drivers/counter/Kconfig.mcux_lpc_rtc

```diff
@@ -1,9 +1,24 @@
-# Copyright (c) 2021, NXP
+# Copyright (c) 2023, NXP
```

Comment:
Copyright should be changed to:
`Copyright 2021,2023 NXP`

Reply:
Thank you, I have update the Copyright.